src/components/HistoryTable/HistoryTableRow/FoldedCommit.tsx

```diff
@@ -34,6 +38,7 @@ export const FoldedCommit = memo(
     numVisibleCols,
     style,
     selected,
+    sendAnalytics,
```

Comment:
any reason we don't wrap all of the analytics logic inside this component i don't think theres any reason to pass it in a conditional prop. I think it makes sense to just define the hook inline in this component.

Reply:
The only reason to pass in an analytics prop is so VariantHistoryRow and TaskHistoryRow can differentiate the analytic event name. If we were to instantiate the sendEvent hook in this file, the only way to differentiate between "task" and "variant" would be to examine the URL which would require switch logic and seems less precise.